go/database/mpt/io/live_test.go

```diff
@@ -287,3 +282,65 @@ func TestCheckEmptyDirectory_FailsIfDirectoryContainsADirectory(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 }
+
+func TestIO_ExportLiveFromArchive(t *testing.T) {
```

Comment:
`...ExportLive...` is a bit misleading, because you export one state from archive, the live db is the always changing head state, not an already sealed state in the archive